src/Entity/SectionAssociation.php

```diff
@@ -16,40 +16,31 @@
  *   bundle_label = @Translation("Section association"),
  *   handlers = {
  *     "access" = "Drupal\workbench_access\SectionAssociationAccessControlHandler",
- *     "storage_schema" = "Drupal\workbench_access\SectionAssociationStorageSchema",
```

Comment:
This was breaking install / uninstall, though I can't figure out why. Perhaps the issue is that we have cardinality which splits the tables into seperate per-field storage.